templates/repo/view_list.tmpl

```diff
@@ -34,6 +34,7 @@
 					</span>
 				{{end}}
 			</th>
+			<th class="text grey"></th>
```

Comment:
?

Reply:
This is just an empty header, because we have added a new `td`